src/docs/product/performance/web-vitals.mdx

```diff
@@ -4,14 +4,14 @@ sidebar_order: 50
 description: "Learn more about Web Vitals in Sentry, and how they give you a better understanding of your application's health."
 ---
 
-[Web Vitals](https://web.dev/vitals/) are a set of metrics defined by Google to measure render time, response time, and layout shift. Each data point provides insights about the overall performance of your application.
+[Web Vitals](https://web.dev/vitals/) are a set of metrics defined by Google to measure render time, response time, and layout shift. Each data point provides insights about the overall <SandboxLink scenario="performance" projectSlug="react">performance</SandboxLink> of your application.
 
-The in-browser Sentry SDKs collect Web Vitals information (where supported) and add that information to frontend transactions. These vitals are then summarized in several graphs for a quick overview of how each frontend transaction is performing for your users.
+The in-browser Sentry SDKs collect <SandboxLink scenario="oneWebVitals" projectSlug="react">Web Vitals</SandboxLink> information (where supported) and add that information to frontend <SandboxLink scenario="oneTransaction" projectSlug="react">transactions</SandboxLink>. These vitals are then summarized in several graphs for a quick overview of how each frontend transaction is performing for your users.
```

Comment:
Same comment here as above. I'd strongly recommend against using the Sandbox link in the words "performance" and "transactions" here.